cmd/state-custom-installer/docker/centos/runCompile.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+cd cli
+git pull
+# git checkout BE-1031-there-is-a-standalone-tool-for-unpacking-and-relocating-artifacts-in-installers 
+go mod vendor
+sudo /state-tool/bin/state run preprocess
+cd ..
+sudo chown -R as-build:as-build cli
+cd cli/cmd/state-custom-installer/docker
+go build -o build/custom-installer github.com/ActiveState/cli/cmd/state-custom-installer
+sudo cp build/custom-installer /custom-installer-data
```

Comment:
Why not just output to `/custom-installer-data` directly if that's where you want it?